### PR TITLE
A bit of tweaking on Action Objects in manpage

### PR DESCRIPTION
**Pull request by mwichmann**
Doc-only change.

Signed-off-by: Mats Wichmann <mats@linux.com>

## Contributor Checklist:

* [ ] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [ ] I have updated `CHANGES.txt` (and read the `README.rst`)
* [X] I have updated the appropriate documentation

### CODE REVIEW

File: doc/man/scons.xml [hunk 1]
```diff
@@ -5436,46 +5433,30 @@ Action('@build $TARGET $SOURCES')
...
-<para>If the first argument is a list,
+<para>If the action argument is a list,
 then a list of Action objects is returned.
 An Action object is created as necessary
 for each element in the list.
 If an element
 <emphasis>within</emphasis>
 the list is itself a list,
-the internal list is the
+the internal list taken as the
```
Comment (bdbaddog) [inline doc/man/scons.xml]: list is taken as ?

File: doc/man/scons.xml [hunk 2]
```diff
@@ -5520,13 +5501,21 @@ def build_it(target=None, source=None, env=None):
...
 <constant>None</constant> is returned.</para>
+  </listitem>
+</itemizedlist>
+
+<para>As usual the environment method form &f-link-env-Action;
+will expand &consvars; in any argument strings, 
+including the action argument, at the time it is called,
+using the construction variables in the &consenv; through which
+it was called. The global function form &f-link-Action;
+delays variable expansion until the Action object is actually used.
```
Comment (bdbaddog) [inline doc/man/scons.xml]: bold or emphasize "delays"?

File: doc/man/scons.xml [hunk 3]
```diff
@@ -5937,20 +5926,18 @@ the entire directory tree
...
 if the entry does not exist.
 Examples:</para>
 
 <programlisting language="python">
 Execute(Delete('/tmp/buildroot'))
 
-env.Command('foo.out', 'foo.in',
-            [Delete('${TARGET.dir}'),
-             MyBuildAction])
+env.Command('foo.out', 'foo.in', [Delete('${TARGET.dir}'), MyBuildAction])
```
Comment (bdbaddog) [inline doc/man/scons.xml]: I almost prefer the previous formatting.. this one breaks the action over two lines...

File: doc/man/scons.xml (hunk 1 above)
Comment (mwichmann) [reply doc/man/scons.xml]: Agree.

File: doc/man/scons.xml (hunk 2 above)
Comment (mwichmann) [reply doc/man/scons.xml]: Sure, why not.

**Pull request merged**